src/shared/components/SVGAxis.tsx

```diff
@@ -88,7 +88,7 @@ export default class SVGAxis extends React.Component<SVGAxisProps, {}> {
             let y:number;
             let transform:string;
             if (this.props.vertical) {
-                x = this.props.x - this.props.tickLength - 20;
+                x = this.props.x - this.props.tickLength - 30;
```

Comment:
This pushes the y-axis label of the mutation diagram further left, so that it doesn't overlap with 4-digit mutation counts.